docs/pull_request_template.md

```diff
@@ -0,0 +1,23 @@
+# Description of Pull Request
+
+Please include a short summary of the proposed changes.
+
+Also include any other relevant information that is useful for this pull request.
+
+Fixes Issue #:
+
+
+## Type of change
+
+Please descirbe the pull request as one of the following: 
```

Comment:
`descirbe` -> `describe`

Reply:
I will handle this simple fix

Reply:
I will handle this simple fix

Reply:
Also, @reibelj can you change `Fixes Issue #:` to `Fixes #Issuenumber`?

Reply:
Yes I can change that as well.